prometheus_node_exporter.yaml

```diff
@@ -0,0 +1,21 @@
+name: prometheus-node-exporter
+version: 0.16
+summary: Prometheus
+description: |
+  Prometheus is a system monitoring tool. Data available on HTTP port 9100.
+architectures: [amd64]
+confinement: strict
+grade: stable
+
+parts:
+  node-exporter:
+    plugin: dump
+    source: https://github.com/prometheus/node_exporter/releases/download/v0.16.0/node_exporter-0.16.0.linux-amd64.tar.gz
```

Comment:
The version here shouldn't be hard-coded, but based on the version specified at the top. I don't know enough about snap packaging to know how this should really work.